apps/guide/src/content/02-installations-and-preparations/04-adding-your-bot-to-servers.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Adding your bot to servers
+category: Installations and preparations
+---
+
+# Adding your bot to servers
+
+After you [set up a bot application](./setting-up-a-bot-application), you'll notice that it's not in any servers yet. So how does that work?
+
+Before you're able to see your bot in your own (or other) servers, you'll need to add it by creating and using a unique invite link using your bot application's client id.
+
+## Bot invite links
+
+The basic version of one such link looks like this:
+
+<CH.Code lineNumbers={false}>
+
+```
+https://discord.com/api/oauth2/authorize?client_id=123456789012345678&permissions=0&scope=bot%20applications.commands
+```
+
+</CH.Code>
+
+The structure of the URL is quite simple:
+
+- _`https://discord.com/api/oauth2/authorize`_ is Discord's standard structure for authorizing an OAuth2 application (such as your bot application) for entry to a Discord server.
+- _`client_id=...`_ is to specify _which_ application you want to authorize. You'll need to replace this part with your client's id to create a valid invite link.
+- _`permissions=...`_ describes what permissions your bot will have on the server you are adding it to.
```

Comment:
```suggestion
- _`permissions=...`_ describes the recommended permissions that your bot would be granted by default upon joining the server.
```

Reply:
Agree with that this needs a wording change since it's not guaranteed your bot will be granted the permissions, but "recommended" would not be the word I'd use either.

"describes the permissions that your bot will request to be granted by default upon joining the server"